stackit/internal/services/argus/instance/resource.go

```diff
@@ -462,6 +486,53 @@ func (r *instanceResource) Schema(_ context.Context, _ resource.SchemaRequest, r
 							},
 						},
 					},
+					"global": schema.SingleNestedAttribute{
+						Description: "Global configuration for the alerts.",
+						Optional:    true,
+						Computed:    true,
+						Attributes: map[string]schema.Attribute{
+							"opsgenie_api_key": schema.StringAttribute{
+								Description: "The API key for OpsGenie.",
+								Computed:    true,
+								Optional:    true,
+							},
+							"opsgenie_api_url": schema.StringAttribute{
+								Description: "The host to send OpsGenie API requests to. Must be a valid URL",
+								Computed:    true,
+								Optional:    true,
+							},
+							"resolve_timeout": schema.StringAttribute{
+								Description: "ResolveTimeout is the default value used by alertmanager if the alert does not include EndsAt, after this time passes it can declare the alert as resolved if it has not been updated. This has no impact on alerts from Prometheus, as they always include EndsAt.",
```

Comment:
```suggestion
								Description: "The default value used by alertmanager if the alert does not include EndsAt. After this time passes, it can declare the alert as resolved if it has not been updated. This has no impact on alerts from Prometheus, as they always include EndsAt.",
```

Reply:
Same for the datasource